integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -74,18 +149,23 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
         managedServerPrefix,
         replicaCount);
 
+    // create ingress rules with path routing for Traefik, Voyager and NGINX
+    createTraefikIngressRoutingRules(domainNamespace);
+    createVoyagerIngressPathRoutingRules();
+    createNginxIngressPathRouting();
+
+    // install WebLogic remote console
+    assertTrue(installAndVerifyWlsRemoteConsole(), "Remote Console installation failed");
   }
 
   /**
-   * Verify WLS Remote Console installation is successful.
    * Verify k8s WebLogic domain is accessible through remote console.
    */
+  @Order(1)
   @Test
   @DisplayName("Verify Connecting to Mii domain through WLS Remote Console is successful")
   public void testWlsRemoteConsoleConnection() {
```

Comment:
Move this method to initAll() to verify the remote console is accessible, so that other test mehod can be run independently with out any specific order.